actions/update-deps/entrypoint.sh

```diff
@@ -72,6 +72,11 @@ if [[ "${FORCE_DEPS}" == "true" ]]; then
   create_pr="true"
 fi
 for x in $(git diff-index --name-only HEAD --); do
+    if find . -type d -name vendor | grep -q vendor; then
```

Comment:
/hold

It should be negated

Reply:
Resolved in https://github.com/knative-extensions/knobots/compare/d2134259e99f7eee88c8441d2b8799dc46482cfa..5f7f494992f4de92810813f886a89665b002b0c8

/unhold